### DISCUSSION TITLE
A Contact import Export feature similar to the roll20 sheet export feature.

### DISCUSSION THREAD

**Discussion by Tyler111762** [chummer5a/chummer5a#4437] [Ideas]
something that would allow you to write up a contact, and share that contact using a string so it can be moved between sheets or shared to other users with greater ease.

**Comment by JoschiZ** [comment]
What is possible right now to import another .chum5 file as connection as far as I know.
Never done it though.

**Comment by ArchonMegalon** [comment]
Is the ability to create a caracter (and save it as chum5-file) and link it as contact not good enough? What is missing?
Do you want better sharing features of characters in general?
